lib/mayaUsd/resources/scripts/mayaUsdLibRegisterStrings.py

```diff
@@ -54,6 +54,8 @@ def mayaUsdLibRegisterStrings():
     register('kLabelStrongerMaterial', 'Stronger than descendants')
     register('kTooltipInheritedStrength', 'This setting cannot be changed on this prim due to the strength setting on an ancestor')
     register('kLabelMaterialNewTab', 'New Tab...')
+    register('kUseOutlinerColorAnn', 'Apply the Outliner color to the display of the prim name in the Outliner.')
+    register('kOutlinerColorAnn', 'The color of the text displayed in the Outliner')
```

Comment:
Very minor: this tooltip does not end with a period.

Reply:
Thanks, I'll fix that.